tasties_app/templates/tasties_app/base.html

```diff
@@ -0,0 +1,97 @@
+{% load static %} {% load bootstrap_icons %}
+<!DOCTYPE html>
+<html lang="en">
+  <!----------------------------------------------------- HEAD ------------------------------------------------------->
+  <head>
+    <!-- Settings: -->
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- Style: -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous"
+    />
+    <script
+        src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.min.js"
+        integrity="sha384-Atwg2Pkwv9vp0ygtn1JAojH0nYbwNJLPhwyoVbhoPwBhjQPR5VtM2+xf0Uwh9KtT"
+        crossorigin="anonymous">
+    </script>
+    <link rel="stylesheet" type="text/css" href="{% static 'css/base.css' %}" />
+    {% block css %}{% endblock %}
+
+    <!-- Name and Description: -->
+    <meta name="description" content="tasties is a platform to share food recipes" />
+    <title>tasties</title>
+  </head>
+  <!----------------------------------------------------- BODY ------------------------------------------------------->
+  <body>
+    <!-- Navbar -->
+    <nav class="navbar navbar-light bg-light">
+
+      <!-- Logo -->
+      <a class="navbar-brand" style="color: #10654e" href="{% url 'index' %}">
+        <img
+          src="{% static 'images/logo.png' %}"
+          width="30"
+          height="25"
+          class="d-inline-block align-top"
+          alt="tasties-logo"
+          style="margin-top: 5px"
```

Comment:
Since you've already created a CSS file to hold all the visual configurations, such as height, width, margin, etc. it's better to have all those configurations there and not in the HTML file